Gemfile

```diff
@@ -54,6 +54,7 @@ group :development do
   gem 'rubocop-rails', '~> 2.3.2', require: false
   gem 'traceroute'
   gem 'web-console', '~> 3.0'
+  gem 'wraith', '4.2.3'
```

Comment:
Why the version pin?

Reply:
Because the current version / version after that is broken for me. (I forget how it was broken but it sure was.)